brave/brave-extension
# Shields popup

test plan
1. `yarn test`
2. `yarn build`
3. `yarn test-e2e`

known issues:

1. advanced controls toggle keeps a white space when toggled. I need to find a decent workaround to reduce popup height (hardcoded in shields pug file) to reduce it. This solves the flickering popup when opening at least
2. shields on/off doesn't respect "ad control" select option

todo for parity with browser-laptop:
1. make reload work
2. edit default settings should open a tab
3. ad control should have "show brave ads"
4. cookie control select (all)
5. https everywhere (stats and toggle)
6. block scripts (stats and toggle)
7. fingerprinting protection (stats and toggle)
8. block phishing/malware (stats and toggle)
